spec/book_spec.rb

```diff
@@ -0,0 +1,48 @@
+require_relative '../book'
+
+RSpec.describe Book do
+  describe '#initialize' do
+    it 'creates a new book instance with the provided publisher and cover state' do
+      publish_date = '2022-01-01'
+      publisher = 'Little, Brown and Company'
+      cover_state = 'bad'
+
+      book = Book.new(publish_date, publisher, cover_state)
+
+      expect(book.publisher).to eq(publisher)
+      expect(book.cover_state).to eq(cover_state)
+    end
+  end
+
+  # describe '#can_be_archived?' do
+  #   it 'returns true when the cover state is bad' do
+  #     publish_date = '2022-01-01'
+  #     publisher = 'Little, Brown and Company'
+  #     cover_state = 'bad'
+
+  #     book = Book.new(publish_date, publisher, cover_state)
+
+  #     expect(book.send(:can_be_archived?)).to eq(true)
+  #   end
+
+  #   it 'returns false when the cover state is not bad' do
+  #     publish_date = '2022-01-01'
+  #     publisher = 'Little, Brown and Company'
+  #     cover_state = 'good'
+
+  #     book = Book.new(publish_date, publisher, cover_state)
+
+  #     expect(book.send(:can_be_archived?)).to eq(false)
+  #   end
+
+  #   it 'returns false when the publish date is less than 5 years ago' do
+  #     publish_date = '2020-01-01'
+  #     publisher = 'Little, Brown and Company'
+  #     cover_state = 'good'
```

Comment:
Hey team, well done all writing sufficient tests 💯👏🏾

[OPTIONAL] I noticed that there's an entire section of tests that are commented on in the book spec. Kindly uncomment these tests since they are testing the `can_be_archived?` public method which is one of the crucial tests for this milestone.

I'm not sure why it was commented out because when I uncommented the code and ran your tests, all the tests pass. There's nothing wrong with it.

Great job!!! 🥂🎆

![image](https://github.com/sumon766/my-things/assets/38088818/c087f166-dc90-47fe-8c8c-dcf4b1c86068)